core/jvm/src/main/scala/scalaz/zio/RTS.scala

```diff
@@ -72,7 +72,7 @@ trait RTS {
    *
    * FIXME: Replace this entirely with the new scheme.
    */
-  final val YieldMaxOpCount = 1048576
+  val YieldMaxOpCount = 1024
```

Comment:
This will have an effect on performance. Have you done benchmarks before / after?

Otherwise, this PR looks great and ready to merge!

Reply:
@jdegoes yep, a couple messages above I posted results https://github.com/scalaz/scalaz-zio/pull/306#issuecomment-432246993 I didn't have the opportunity to run all 20 forks, but nothing seemed crazy off, so I assumed it's a safe change.

Reply:
For the context - @regiskuckaertz suggested to change MaxOpCount to improve the situation in this scenario https://github.com/scalaz/scalaz-zio/issues/273